aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -251,6 +253,29 @@ public void testDeleteTableWithoutPurge() {
   public void testDeleteTableWithPurge() {
     String namespace = createNamespace();
     String tableName = createTable(namespace);
+    Table table = glueCatalog.loadTable(TableIdentifier.of(namespace, tableName));
+
+    final DataFile testFile =
```

Comment:
nit: We don't really use `final` with temp vars in Iceberg.
I know it is debatable but we've agreed it is not worth is.

Reply:
Oh yeah, I've updated it!